src/placement/jump_map.c

```diff
@@ -554,6 +565,18 @@ get_object_layout(struct pl_jump_map *jmap, struct pl_obj_layout *layout,
 
 
 	for (i = 0; i < jmop->jmop_grp_nr; i++) {
+
+		if (doms_left < jmop->jmop_grp_size) {
+			uint32_t start_dom;
+			uint32_t end_dom;
+
+			doms_left = jmap->jmp_domain_nr;
+			start_dom =  min_redundant_dom - root;
+			end_dom = start_dom + (doms_left - 1);
```

Comment:
I think we'd better use consistent variable names:
e.g.
doms_left, dom_start, dom_end